src/levanter/compat/torch_serialization.py

```diff
@@ -151,6 +151,9 @@ def default_eqx_module_from_state_dict(mod: Mod, state_dict: StateDict, prefix:
         value = getattr(mod, field.name)
         # TODO: might want to add a flag that allows missing keys?
         new = jax_tree_from_state_dict(value, state_dict, apply_prefix(prefix, key))
+        # Do not try to update parameters that are never defined
+        if not value and not new:
```

Comment:
this should be an explicit check again None yeah?

Reply:
Yes! Done.